ovh-ux/manager
# test(freefax): add standalone tests

## test(freefax): add standalone tests

### Description of the Change

03c6fdd — test(freefax): add standalone tests

* add test with navigation between states in freefax modules
* add tests to check if translations are present when used (`data-translate`and `data-translate-attr` directives and `$translate.instant` call)
* Some `data-e2e-id` in html for tests purpose (see [Cypress Best Practices](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements))
* add missing translations
